src/app/globals.css

```diff
@@ -37,4 +37,19 @@
     @apply h-full w-full overflow-hidden bg-background text-foreground transition-colors duration-200;
   }
 
+}
+
+
+@keyframes pie-graph-loading {
```

Comment:
Can't rely on global styling in scope and with non-globally unqiue naming.
This file is actually not for the library itself but for documentation website.
Components should generally have isolated styles as a best-practice otherwise coupling is created.

To avoid this can ethier do:

1. "https://tailwindcss.com/docs/animation#pulse" Try this first which is a inbuilt tailwind keyframes animation **similar** to what we want, if this gets close enough let's just use this.
2. Fallback to .module.scss (css modules)
```
// Pie.module.scss

.loading {
     animation-name: pie-graph-loading;
     &--dark {
         animation-name: dark-pie-graph-loading;
     }
}
@keyframes pie-graph-loading {
	0% { fill: #a7a7a7; }
	30% { fill: #cdcdcd; }
	60% { fill: #a7a7a7; }
	100% { fill: #a7a7a7; }
}

@keyframes dark-pie-graph-loading {
	0% { fill: #2d2d2d; }
	30% { fill: #3c3c3c; }
	60% { fill: #2d2d2d; }
	100% { fill: #2d2d2d; }
}
```

and Pie.tsx
```
// Pie.tsx
import styles from "./PieGraph.module.scss";
... later

// something like this.
className={cx("tailwind class", "tailwindclass", styles.lioading)
```
3. Try find a tailwind only approach or with svg elements this would be best. You can see what @katesheldon18 has done for Pie's empty state to get light/dark

Reply:
Tried 3. for a while before but didn't think about the filter property that @katesheldon18 used and that seemed to work fine!